js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,13 @@
+function lastIndexOf(...args) {
+  let searchedValue = args[0];
```

Comment:
"searchedValue" can be renamed to something more meaningful like "valueToSearch".